toolkit/utils/db-sync-sqlx/src/lib.rs

```diff
@@ -174,5 +223,8 @@ impl From<MainchainAddress> for Address {
 	}
 }
 
+/// Cardano epoch nonce, ie. random 32 bytes generated every epoch during committee selection.
```

Comment:
I'm not sure if Cardano has "committee selection".
```suggestion
/// Cardano epoch nonce, ie. random 32 bytes generated by Cardano for every epoch.
```

Reply:
it doesn't, it has random leader selection. fixed